instaseis/scripts/repack_db.py

```diff
@@ -490,7 +490,11 @@ def repack_database(input_folder, output_folder, contiguous,
 
     assert found_filenames, "No files named `ordered_output.nc4` found."
 
-    os.makedirs(output_folder)
+    if input_folder == output_folder:
```

Comment:
A minor nitpick and a bit of a constructed case but why not catch it: `input_folder` and `output_folder` might be different strings but refer to the same folder. Using

```python
input_folder = os.path.normpath(os.path.realpath(input_folder))
output_folder = os.path.normpath(os.path.realpath(output_folder))
if input_folder == output_folder:
    ...
```

would solve this.